src/components/Dashboard/FilingHistoryList/HeaderActions.vue

```diff
@@ -131,7 +130,8 @@ export default class HeaderActions extends Mixins(AllowableActionsMixin) {
 
       case FilingTypes.CHANGE_OF_ADDRESS:
       case FilingTypes.CHANGE_OF_DIRECTORS:
-        if (this.isBenBcCccUlc) {
+      case FilingTypes.SPECIAL_RESOLUTION:
+        if (this.isBenBcCccUlc || this.isCoop) {
```

Comment:
This will allow Coop COA corrections and COD corrections to route to Edit UI (instead of being local corrections). Is that the intent?

Reply:
Yes, coop special resolution corrections will taken by business-edit-ui

Reply:
My question is whether coop COA corrections and COD corrections will be taken by Edit UI.

Reply:
oh sorry I misunderstood, no, I'll update it.

Reply:
I've added a new conversation for this. Resolving this one.